src/benchmarklib/benchmark_runner.cpp

```diff
@@ -7,19 +7,18 @@
 #include <boost/accumulators/accumulators.hpp>
 #include <boost/accumulators/statistics/mean.hpp>
 #include <boost/accumulators/statistics/stats.hpp>
-
 #include <boost/algorithm/string/join.hpp>
 #include <boost/range/adaptors.hpp>
-#include "cxxopts.hpp"
+#include <cxxopts.hpp>
```

Comment:
Actually, I think boost's usage of `<>` is rather wrong. For `cxxopts` it is most certainly wrong as it is not a system library.

We might be better of pulling boost as a third party library (different boost version on Ubuntu/MacOS brew can be a pain). Then the entire discussio would be rather simple.

Reply:
Just grepped a few variations, seems we usually import magic_enum and boost with `<>` and only in a few cases with `""`. Totally fine to change this (altogether with the Boost <1.71 workaround). However, I'd move this to a new issue/PR if that's okay for you.

Reply:
Taken from the [Boost wiki](https://github.com/boostorg/wiki/wiki/Getting-Started%3A-Overview#installing-boost):
> The initial cloning will take at least 45 minutes on a 3.0 mbps Internet connection and will consume roughly 1.5 GB of disk space. Cloning is only done once, so this lengthy initial time is not a long-term concern.

Are we sure we want this to be a third party library (CI builds, etc.)?

Reply:
Well, that might be a good reason to keep it as a sys library. :D

Reply:
At least the <1.71 workaround is now fixed 🤷‍♀️